vitals/BatteryValidator.java

```diff
@@ -0,0 +1,47 @@
+package vitals;
+
+public class BatteryValidator {
+	static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
+		return isTemperatureOk(temperature) && isSOCOk(soc) && isChargeRateOk(chargeRate);
+	}
+
+	public static boolean isTemperatureOk(float temperature) {
```

Comment:
why don't you parameterize isLower and isHigher and Print ErrorMessage functions an an argument to is*******ok functions